torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1478,10 +1478,20 @@ def call_function(
 
 
 class WrapHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def install_subgraph_in_output_graph(
```

Comment:
Wrap is the base for some other HOPs too, right? (like checkpoint?).

We should probably give it a better name at some point. "Wrap" is basically a HOP that ends up executing the subgraph you give it without changing the calling convention of that subgraph. Elias was also building a HOP on top of Wrap (called invoke_quant)